scripts/add-unreleased-section.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+die() {
+  local exit_code=1
+  local OPTIND=1
+  local opt
+
+  while getopts "c:" opt; do
+    case "$opt" in
+      c)
+        exit_code="$OPTARG"
+        ;;
+    esac
+  done
+
+  shift $((OPTIND - 1))
+
+  echo "ERROR:" "$@" >&2
+  exit $exit_code
+}
+
+print_usage() {
+  cat <<EOF
+Usage: ${0##*/} <package>
+       ${0##*/} -h
+Adds an unreleased section to the CHANGELOG.md for <package>.
+
+OPTIONS
+EOF
+  sed -n '/^[[:space:]]*###/ s//   /p' "$BASH_SOURCE"
+}
+
+while getopts :h opt; do
+  case "$opt" in
+    ### -h	Show this help
+    h)
+      print_usage
+      exit 0
+      ;;
+    \?)
+      die -c 2 "Invalid option: -${OPTARG}"
+      ;;
+  esac
+done
+
+shift $((OPTIND - 1))
+
+cd "${BASH_SOURCE%/*}/.."
+
+package="$1"
+changelog="packages/@storefront/${1}/CHANGELOG.md"
```

Comment:
You should use the package variable instead of ${1} when setting the changelog variable.